networkx/linalg/algebraicconnectivity.py

```diff
@@ -585,3 +590,73 @@ def spectral_ordering(
             order.extend(component)
 
     return order
+
+
+def spectral_bisection(
+    G, weight="weight", normalized=False, tol=1e-8, method="tracemin_pcg", seed=None
```

Comment:
AFAICT, the `tol`, `method` and `seed` parameters are all unused in the implementation. Can we remove them here? Or are these params that need to be forwarded to `fiedler_vector`?